examples/ebpf-profiler/generate-traffic.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+while true; do
+	curl -s http://java:8080/rolldice || echo "error reaching java service"
+	curl -s http://go:8081/rolldice || echo "error reaching go service"
+	curl -s http://python:8082/rolldice || echo "error reaching python service"
+	curl -s http://dotnet:8083/rolldice || echo "error reaching dotnet service"
+	curl -s http://nodejs:8084/rolldice?rolls=5 || echo "error reaching nodejs service"
```

Comment:
Just curious, why does the Node.js example have a query string parameter when the others don't?

Reply:
didn't look into it - just copied from https://github.com/grafana/docker-otel-lgtm/blob/d77e557e098e04bd328ad360001595e2d6bd0326/generate-traffic.sh#L7